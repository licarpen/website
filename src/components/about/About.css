.About{
  width: 80%;
  margin: auto;
  margin-top: 25px;
  border-radius: 0.2em;
  background: rgb(68, 67, 67);
  padding: 25px;

  img{
    display: block;
    width: 60%;
    margin: auto;
    margin-bottom: 20px;
    border: 3px white solid;
  }

  p{
    text-align: left;
    font-size: 20px;
    line-height: 24px;
  }

  @media screen and (min-width: 1000px){
    width: 35%;
    img{
      width: 25%;
      float: left;
      margin-right: 30px;
      margin-bottom: 0px;
    }
    p{
      font-size: 20px;
      line-height: 26px;
    }
  }
}