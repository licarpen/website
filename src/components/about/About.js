import React from 'react';
import styles from './About.css';
import me from '../../assets/me.png';

const About = () => (
  <div className={styles.About}>
    <img src={me}></img>
    <p>Lisa Carpenter is a software engineer based in Portland, Oregon.  She absolutely loves to code.  Her expertise in physics and mathematics enables her to craft logical, eloquent solutions to complex problems.  She writes readable, tested code and is in search of an incredible team.</p>
    <br/>
    <p>When not coding up a storm, Lisa can be found playing jazz trumpet, advocating for bike/ped/scooter infrastructure, sharing physics demonstrations with friends and family, and exploring the outdoors through rock climbing, hiking, and biking.</p>
  </div>
);


export default About;
