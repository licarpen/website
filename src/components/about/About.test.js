import React from 'react';
import { shallow } from 'enzyme';
import About from './About';

describe('About component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<About />);
    expect(wrapper).toMatchSnapshot();
  });
});
