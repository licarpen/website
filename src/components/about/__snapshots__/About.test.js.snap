// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component matches a snapshot 1`] = `
<div
  className="About"
>
  <p>
    Lisa Carpenter is a software engineer based in Portland, Oregon.  She absolutely loves to code.  Her expertise in physics and mathematics enables her to craft logical, eloquent solutions to complex problems.  She writes readable, tested code and is in search of an incredible team.
  </p>
  <br />
  <p>
    When not coding up a storm, Lisa can be found playing jazz trumpet, advocating for bike/ped/scooter infrastructure, sharing physics demonstrations with friends and family, and exploring the outdoors through rock climbing, hiking, and biking.
  </p>
</div>
`;
